src/components/chat/feedbackButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import { faThumbsUp, faThumbsDown } from '@fortawesome/free-regular-svg-icons';
+import {
+  faThumbsUp as faThumbsUpSolid,
+  faThumbsDown as faThumbsDownSolid,
+} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { Feedback } from '@/models/chat';
+
+import AnimatedButton from '../ui/animatedButton';
+
+interface FeedbackButtonsProps {
+  feedback: Feedback | null;
```

Comment:
in general I tend to do `feedback?: Feedback` instead of `feedback: Feedback | null`.  Any reason you prefer null instead of undefined for this kind of thing?  I know they are pretty similar but useState defaults to undefined and syntax is prettier so i tend to go with the former

Reply:
i do `feedback?: Feedback` when it is an optional prop, and `feedback: Feedback | null` when i want the component to always be passed it, even if it's null. for the first one you can do:
```js
  <FeedbackButtons
    //   feedback={feedbackSent}
    onFeedback={onFeedback}
    disableFeedback={feedbackSent !== null}
  />
  ```
  whereas with the `Feedback | null`, it'll throw an error. basically an idiot-proofing step and i think makes it clearer when things are optional vs nullable. i don't always bother to make the distinction, but with this being the `FeedbackButtons` and the `feedback` prop controlling the state of the buttons, i want it to always be required